tests/framework/microvm.py

```diff
@@ -620,6 +620,7 @@ def basic_config(
         track_dirty_pages: bool = False,
         rootfs_io_engine=None,
         cpu_template: Optional[str] = None,
+        enable_entropy_device = False,
```

Comment:
Are there any downsides to having it enabled by default?

Reply:
Not sure if any of the tests makes assumption about which devices are enabled or not. I was thinking snapshotting tests, etc.

Reply:
Ok... my thinking is: we would expect customers to enable this most of the time. Also as a nice side-effect, faster boots could mean faster test runs :)

Reply:
It makes sense, yes. But from the point of view of testing, I prefer limiting the scope of the test to what is needed for the purposes of the test. IMHO, it leads to more stable tests.

Reply:
not strong opinions though.

Reply:
if we ask customers to enable this then it makes sense to keep it enabled by default and to check if this has any effect on performance we could dry run AB test on it. If AB test doesn't show any regression with it being on by default then I guess it would be good to have enabled by default.

Reply:
That's what this PR does. It enables it for boot-time performance tests. I just made it optional for any other test, since it is not strictly needed for testing, say block or net devices.